src/Entities/CassetteBlocks/ManualCassetteController.cs

```diff
@@ -13,13 +14,29 @@ public class ManualCassetteController : AbstractInputController
     private int roomBeats;
     private int currentIndex;
 
+    private string blueFlag;
```

Comment:
These can be declared as constants strings, instead of private fields initialized in the constructor.
```suggestion
    private const string blueFlag = "CH_cas_blue";
```
Then you don't have to add those lines in the constructor (26 to 30).